server/db/scorepads.js

```diff
@@ -28,8 +28,8 @@ function initScorepad(scorepad, players) {
  *
  * @param {function} onResult callback that is executed with the result documents from the database
  */
-function get(onResult) {
-  scorepadModel.find({}, (scorepads) => {
+function get(game, onResult) {
```

Comment:
Mit der Änderung die ich oben vorgeschlagen habe, sollte hier dann `query` stehen:
```suggestion
function get(query, onResult) {
```

Reply:
Bei der Funktion fehlt noch ein `@param {Object} query` mit Beschreibung für die Dokumentation der Funktion - wäre super, wenn wir das aktuell halten können :slightly_smiling_face: